src/main/kotlin/org/jetbrains/bio/genome/Annotations.kt

```diff
@@ -103,6 +119,67 @@ object Repeats {
 
         return builder.build()
     }
+
+    private fun parseHs1(
+        repeatsPath: Path,
+        chromosomes: Map<String, Chromosome>,
+    ): ImmutableListMultimap<Chromosome, Repeat> {
+        val builder = ImmutableListMultimap.builder<Chromosome, Repeat>()
+
+        val reader = InputStreamReader(GZIPInputStream(FileInputStream(repeatsPath.toFile())))
+        BufferedReader(reader).lines()
+            .filter { isValidHs1RepeatsLine(it) }
+            .forEach {
+                val parsingResult = parseHs1RepeatsLine(it, chromosomes)
+                if (parsingResult != null) builder.put(parsingResult.first, parsingResult.second)
+            }
+
+        return builder.build()
+    }
+
+    private fun isValidHs1RepeatsLine(line: String): Boolean {
+        val substringPresentOnlyInHeader = " repeat "
+        val chrMarker = "chr"
+
+        return line.isNotBlank() && line.contains(chrMarker) && !line.contains(substringPresentOnlyInHeader)
+    }
+
+    private fun parseHs1RepeatsLine(line: String, chromosomes: Map<String, Chromosome>): Pair<Chromosome, Repeat>? {
```

Comment:
Please add a test with a short prepared test data for repeats in hs1 specific format.

Reply:
I've added some tests for hs1 repeats parsing.